src/components/calculator/Calculator.js

```diff
@@ -0,0 +1,90 @@
+import './Calculator.css';
+
+const Calculator = () => (
+  <section className="calculator">
+    <input className="calculator__output calculator__padding" />
+    <button type="button" className="calculator__key calculator__padding">
+      AC
```

Comment:
- [ ] The Calculator.js component should be class-based as per the project's requirement. Kindly refactor it to the class-based component. :pray: